docs/guides/features/command.md

```diff
@@ -1,4 +1,4 @@
-# Calling Rust from the frontend
+# Communication between Rust and the Frontend
```

Comment:
```suggestion
# Calling Rust from the frontend
```

Please revert this for now as it doesn't look good in the sidebar since it's the only item taking up multiple lines:
![grafik](https://user-images.githubusercontent.com/30730186/232746267-359644e8-f5bf-4882-8e30-b31a71f398a8.png)

We could revisit this in the new docs in the future.

Reply:
Yep, didn't notice that, thanks. Changing title back now...

Reply:
Committed!